src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -515,7 +515,9 @@ object DeepDiveLogCompiler extends DeepDiveLogHandler {
           inputQueries += s"""SELECT DISTINCT ${ headStr }, 0 AS id, ${labelCol} AS label
           ${ qc.generateSQLBody(cqBody) }
           """
-        } else if ((ss.schemaDeclarationGroupByHead contains stmt.q.head.name) && (ss.schemaDeclarationGroupByHead(stmt.q.head.name)(0).isQuery) && (stmt.q.head.name startsWith "dd_new_")) {
+        } else if ((ss.schemaDeclarationGroupByHead contains stmt.headName) &&
+          (ss.schemaDeclarationGroupByHead(stmt.headName)(0).isQuery) &&
+          (stmt.headName startsWith "dd_new_")) {
```

Comment:
Sorry, I now feel like asking too much clean up, for code you just introduced line breaks, but it's pretty painful to decipher what each condition means.  Would have been much nicer if they were like:
```scala
if (ss.hasSchemaDeclared(stmt) &&
 ss.isQueryTerm(stmt.headName) &&
 ss.isForIncremental(stmt)) {
```